quarkusissue.java

```diff
@@ -43,6 +40,8 @@
     @Option(names={ "-c", "--columns"}, description = "Columns to use for console rendering.")
     Integer columns;
 
+    @Option(names={ "-m", "--print-markdown"}, description = "Prints markdow to standard output instead of copying it to clipboard.")
```

Comment:
```suggestion
    @Option(names={ "-m", "--print-markdown"}, description = "Prints markdown to standard output instead of copying it to clipboard.")
```